assets/src/features/shared/components/BlockingScreen.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import RotateRight from "../../room-page/icons/RotateRight";
+
+const BlockingScreen: React.FC<{ message?: string }> = ({ message }) => {
+  return (
+    <div
+      className="absolute inset-0 z-30 flex h-full w-full flex-col items-center justify-center gap-y-4 bg-brand-sea-blue-100 font-aktivGrotesk text-brand-dark-blue-500"
```

Comment:
Seems to be a nitpick (i.e. don't waste your time if its not a trivial fix), but when I'm hiding browser's address bar, the viewport or something changes and I'm able to see a part of the page below, here's a video of that:

https://user-images.githubusercontent.com/2119556/219687603-698064ff-9d43-40b3-a4b1-152b5354e753.mp4